package.json

```diff
@@ -43,14 +43,14 @@
     "refmterr": "^3.3.0"
   },
   "devDependencies": {
-    "ocaml": "4.8.x",
-    "@opam/dune": "2.0.1",
+    "ocaml": "4.9.x",
```

Comment:
Should I update the `ocaml` dependencies as well? i.e: from `4.6.10 - 4.8` to `4.6.10 - 4.9` ?

Reply:
Yes, let's do that.

Reply:
done